infra/helper.py

```diff
@@ -46,6 +48,14 @@
 if sys.version_info[0] >= 3:
     raw_input = input
 
+CORPUS_URL_FORMAT = (
+    'gs://{project_name}-corpus.clusterfuzz-external.appspot.com/libFuzzer/'
+    '{project_name}_{fuzz_target}/')
```

Comment:
I've noticed that this doesn't always work. Some projects do not have `$project_name` suffix, while some others do. E.g. for systemd this prefix is needed, while for zlib it's not. Will investigate on Monday.